pkg/controller/controller.go

```diff
@@ -529,6 +529,10 @@ func (p *csiProvisioner) getVolumeContentSource(options controller.VolumeOptions
 	}
 	glog.V(5).Infof("VolumeSnapshotContent %+v", snapContentObj)
 
+	if snapContentObj.Spec.VolumeSnapshotSource.CSI == nil || len(snapContentObj.Spec.VolumeSnapshotSource.CSI.SnapshotHandle) == 0 {
```

Comment:
Just to confirm we don't need to check Spec.VolumeSnapshotSource because it is required field?

Reply:
So I removed the check for SnapshotHandle but kept the check for CSI is nil.  Let me know if this is okay.